modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologydiversification/v0.1.3.json

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/morphologydiversification/v0.1.3/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.4"
+  ],
+  "shapes": [
+    {
+      "@id": "this:MorphologyDiversificationShape",
+      "@type": "sh:NodeShape",
+      "label": "Morphology diversification shape",
+      "comment": "Morphology diversification: repair, scaling, grafting, mix-and-match, cloning.",
+      "targetClass": "nsg:MorphologyDiversification",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.4/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Morphology diversification used entities",
+              "description": "Morphology diversification used entities.",
+              "minCount": 1
```

Comment:
I would add a maxCount and set it to 2. I guess you don't want to have more than two used values.

Reply:
No, it can be more than two entities, depending which diversification stages where enabled.

Reply:
Okay.